tests_v2/test_page_cli_socket_parity.py

```
@@ -0,0 +1,279 @@
+import json
+import os
+import subprocess
+import sys
+from pathlib import Path
+from typing import Dict, List, Tuple
+
+sys.path.insert(0, str(Path(__file__).parent))
+from cmux import cmux, cmuxError
+
+
+SOCKET_PATH = os.environ.get("CMUX_SOCKET", "/tmp/cmux-debug.sock")
```

Ruff (0.15.14)

[error] Probable insecure usage of temporary file or directory: "/tmp/cmux-debug.sock"

(S108)